base_export_manager/models/ir_model_access.py

```diff
@@ -8,4 +8,6 @@
 class IrModelAccess(models.Model):
     _inherit = "ir.model.access"
 
+    # WARN: This can't be used in 'check()'
+    # See 'base > ir_model > line 1530
```

Comment:
:stop_sign: This comment is tricky. If a new commit enters that file, the line number could change.

It's better if you provide an absolute link to the code, from github.